flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/agg/WindowAggregateTest.scala

```diff
@@ -85,6 +85,18 @@ class WindowAggregateTest extends TableTestBase {
     util.verifyPlanNotExpected(sql, "TUMBLE(rowtime")
   }
 
+  @Test
+  def testWindowWrongWindowParameter(): Unit = {
+    expectedException.expect(classOf[TableException])
+    expectedException.expectMessage(
+      "Only constant window intervals with millisecond resolution are supported")
+
+    val sqlQuery =
+      "SELECT COUNT(*) FROM MyTable GROUP BY TUMBLE(proctime, INTERVAL '2-10' YEAR TO MONTH)"
```

Comment:
Test for `INTERVAL '1' MONTH`? This is the minimal value of `INTERVAL_YEAR_MONTH` and more often be used.

Reply:
Add one more test `INTERVAL '35' DAYS` which should work?

Reply:
> Add one more test `INTERVAL '35' DAYS` which should work?

yes, it's valid